packages/react-menu/tsconfig.json

```diff
@@ -1,23 +1,16 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "baseUrl": ".",
+    "target": "ES5",
+    "lib": ["ES5", "dom"],
     "outDir": "dist",
-    "target": "es5",
-    "module": "commonjs",
     "jsx": "react",
     "declaration": true,
-    "sourceMap": true,
     "experimentalDecorators": true,
```

Comment:
Pretty sure this package does not need this option (should only be needed for packages which include legacy stuff)